crates/workspace/src/pane.rs

```diff
@@ -1786,14 +1795,36 @@ impl Pane {
                                     });
                                 }),
                             )
-                            .when_some(parent_abs_path, |menu, abs_path| {
+                            .when_some(rel_path, |menu, path| {
+                                menu.entry(
+                                    "Copy Path",
```

Comment:
```suggestion
                                    "Copy Relative Path",
```
Like vscode IIRC ?

Reply:
I considered that, but I'm not sure it's always correct if a file was opened with an absolute path to begin with. What does vs code do in that case?

Reply:
> I considered that, but I'm not sure it's always correct if a file was opened with an absolute path to begin with. What does vs code do in that case?

That's true, what can we do if the user opens files out of the workspace.
IIRC it is possible in VScode but not in Zed for now.

Reply:
Ah good point, nevermind then.